src/main/java/com/khoros/twitter/TwitterLabApplication.java

```diff
@@ -0,0 +1,28 @@
+package com.khoros.twitter;
+
+import com.khoros.twitter.resources.GetTimelineResource;
+import com.khoros.twitter.resources.PostTweetResource;
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+
+public class TwitterLabApplication extends Application<TwitterLabConfiguration> {
+
+    public static void main(String[] args) throws Exception {
+        new TwitterLabApplication().run(args);
+    }
+
+    @Override
+    public void initialize(Bootstrap<TwitterLabConfiguration> bootstrap) {
```

Comment:
do you need this?

Reply:
initialize? i think it needs it to extend Application